charts/umbrella/values.yaml

```diff
@@ -1321,3 +1324,50 @@ bdrs-server-memory:
         className: "nginx"
         tls:
           enabled: false
+
+ssi-dim-wallet-stub:
+  enabled: false
+  wallet:
+    replicaCount: 1
+    host: ssi-dim-wallet-stub.tx.test
+    nameSpace: "umbrella"
+    appName: "ssi-dim-wallet-stub"
+    configName: "ssi-dim-wallet-config"
+    serviceName: "ssi-dim-wallet-service"
+    secretName: "ssi-dim-wallet-secret"
+    ingressName: "ssi-dim-wallet-ingress"
+    image:
+      repository: tractusx/ssi-dim-wallet-stub
+      pullPolicy: IfNotPresent
+      tag: "0.0.2"
```

Comment:
Due to updating the chart version in the Chart.yaml this image setting shouldn't be needed any longer as [the image in the deployment default to the appVersion in the Chart.yaml](https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/blob/ssi-dim-wallet-stub-0.1.2/charts/ssi-dim-wallet-stub/templates/deployment.yaml#L44)
```suggestion
```